src/openforms/registrations/contrib/objects_api/tests/test_template.py

```diff
@@ -38,15 +38,17 @@ def test_default_template(self, m):
         )
         SubmissionFileAttachmentFactory.create(submission_step=submission.steps[0])
         config = ObjectsAPIConfig(
-            objects_service=ServiceFactory.build(
+            productaanvraag_type="terugbelnotitie",
+        )
+        config_group = ObjectsAPIGroupConfig.objects.create(
```

Comment:
please define a factory so that we can cut down on boilerplate a lot :) this should be possible, leading to an equally valid configuration:

```py
config_group = ObjectsAPIGroupConfigFactory.create(
    objects_service__api_root="https://objecten.nl/api/v1/",
    drc_service__api_root="https://documenten.nl/api/v1/",
)
```